test/whl-scale-tests/crash-dumps/main.ps1

```diff
@@ -0,0 +1,49 @@
+<#
+.SYNOPSIS
+Generate Crash Dumps
+.DESCRIPTION
+Uses Azure watson's hugedump exe to generate crash dumps. Requires the following Env vars are set:
+* CRASHD_NUM_CRASHES - Number of crash dumps to generate
+* CRASHD_ALLOC_STRATEGY - Strategy to use for memory allocation. Options are:
+      va : allocate using VirtualAlloc
+      ha : allocate using HeapAlloc
+      sa : allocate on threads' stacks
```

Comment:
This clarification belongs in the configmap as well

Reply:
After talking with Chris offline we have decided to document this inside the md file for clarity.